src/ServiceControlInstaller.PowerShell/Cmdlets/ServiceControlInstances/NewServiceControlUnattendedFile.cs

```diff
@@ -1,152 +0,0 @@
-// ReSharper disable UnassignedField.Global
-// ReSharper disable MemberCanBePrivate.Global
-// ReSharper disable UnusedAutoPropertyAccessor.Global
-
-namespace ServiceControlInstaller.PowerShell
-{
-    using System;
-    using System.Linq;
-    using System.Management.Automation;
-    using Engine.Configuration.ServiceControl;
-    using Engine.Instances;
-
-    [Cmdlet(VerbsCommon.New, "ServiceControlUnattendedFile")]
-    public class NewServiceControlUnattendedFile : PSCmdlet
```

Comment:
@mikeminutillo we looked back in history and saw that the deprecation warning was added in 2017 so we figured it is about time to remove this code. Do you see any problems with that? Anything we missed?